docs/deployment.md

```diff
@@ -0,0 +1,270 @@
+<!--
+SPDX-FileCopyrightText: 2021 Open Networking Foundation <info@opennetworking.org>
+SPDX-License-Identifier: Apache-2.0
+
+-->
+
+# gNBSim deployment options
+
+## Option1: As docker container 
+
+[Aether onRamp](https://docs.aetherproject.org/master/onramp/gnbsim.html) installs gNBSim as docker container next to SD-Core (5G Core).
+If user wish to use just gNBSim and test it with some other 5G Core then Aether onRamp allows just installation of gNBSim. 
+Aether onRamp makefiles are readable and  user can achieve the required goal.
+
+## Option2: As standalone executable
+
+If you want to run gNBSim as a standalone running executable tool then follow below steps.
+
+        $ git clone git@github.com:omec-project/gnbsim.git
```

Comment:
Any reason you prefer `git`? why not `https`? I think using https is more general/typical because there may be cases where `git` is blocked in corporate networks
```suggestion
        $ git clone https://github.com/omec-project/gnbsim.git
```